### DISCUSSION TITLE
Support transitive dependency information for java container image scans

### DISCUSSION THREAD

**Discussion by theakshaymenon** [aquasecurity/trivy#5469] [Ideas]
### Description

Is there a plan to support `Indirect` and `DependsOn` fields for JAR/WAR/PAR/EAR files, similar to how we handle them for pom.xml?

### Target

Container Image

### Scanner

None

**Comment by DmitriyLewen** [comment]
Hello @theakshaymenon 
Created #5486 for this task.
